e2e/cypress/support/okta_commands.js

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {_} from 'lodash';
+import * as TIMEOUTS from '../fixtures/timeouts';
+
+Cypress.Commands.add('oktaGetApp', () => {
+    return cy.request({
+        method: 'GET',
+        url: Cypress.env('okta_api_url') + '/apps/' + Cypress.env('okta_mm_app_instance_id'),
+        headers: {
+            Authorization: 'SSWS ' + Cypress.env('okta_mm_app_token'),
+        }
+    }).then((response) => {
+        expect(response.status).to.be.equal(200);
+        //console.log('oktaGetApp ' + JSON.stringify(response.body));
+        return cy.wrap(response.body);
+    });
+});
+
+
+Cypress.Commands.add('oktaUpdateAppSettings', (appSettingsJson) => {
+    return cy.request({
+        method: 'PUT',
+        url: Cypress.env('okta_api_url') + '/apps/' + Cypress.env('okta_mm_app_instance_id'),
+        headers: {
+            Authorization: 'SSWS ' + Cypress.env('okta_mm_app_token'),
+        },
+        body: appSettingsJson
+    }).then((response) => {
+        expect(response.status).to.equal(200);
+        //console.log('oktaUpdateAppSettings ' + JSON.stringify(response.body));
+        return cy.wrap(response.body);
+    });
+});
+
+function buildProfile(user) {
+    const profile = {
+        firstName: user.firstname === null ? null : user.firstname,
+        lastName: user.lastname === null ? null : user.lastname,
+        email: user.email === null ? null : user.email,
+        login: user.email === null ? null : user.email,
+        userType: user.userType === null ? null : user.userType
+    };
+    return profile;
+}
+
+Cypress.Commands.add('oktaCreateUser', (user = {}) => {
+    const profile = buildProfile(user);
+    return cy.request({
```

Comment:
For commands that's not part of usual UI flow (like login / logout), I suggest to use `axios` instead of `cy.request` so that cookies are not received/sent after use.  Doing so,  there are places that you might not need to do  `cy.oktaDeleteSession`.  Commands included are probably those using `okta_mm_app_token`.  See `external_request.js` for similar pattern.